app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -97,6 +94,25 @@ class NewConversationViewModel
         }
     }
 
+    private suspend fun tryGetAllContacts() {
```

Comment:
I think getAllContacts better name than tryGet all contacts
or just all contacts so it won't conflict with the use case name